docs/5 IBC and Relayer/2 Scaffold.md

```diff
@@ -0,0 +1,15 @@
+# IBC Scaffold
+
+Starport supports IBC-specifc scaffolding.
+
+Creating a Cosmos SDK module with IBC logic:
+
+```
+starport module create ibcdex --ibc
+```
+
+Creating a custom packet:
```

Comment:
```suggestion
To create a custom packet:
```

Reply:
again, narrative for the example is handy for learners like me

Reply:
I agree. We have a separate section for both module scaffolding and packet scaffolding. I don't want to duplicate content. This section can be used as a guide if you want to know more about IBC-related scaffolding.